learning-development/accessibility.qmd

```diff
@@ -50,6 +49,115 @@ For more advice about colour in charts and visualisations specifically, includin
+- Compliance: These tools help ensure that your spreadsheets meet legal accessibility requirements without extensive manual adjustments, reducing the risk of non-compliance.
+
+By incorporating tools like ``gptables`` or ``a11ytables`` into your coding workflow, you can enhance the accessibility of your spreadsheets with minimal extra effort, making your data more inclusive and easier to use for everyone.
+
+#### Producing accessible tables in R using [``a11ytables``](https://co-analysis.github.io/a11ytables/#:~:text=%7Ba11ytables%7D%201%20Purpose%20An%20R%20package%20to%20help,3%20Contribute%20...%204%20Install%20...%20More%20items):
+
+- Overview: The ``a11ytables`` package in R serves a similar purpose by helping users create accessible tables within their spreadsheets. The package includes functions that facilitate the creation of tables with correct headers, descriptive text, and appropriate structures, making them accessible to all users, including those relying on assistive technologies.
+- Benefits: Using ``a11ytables`` ensures that your spreadsheets are automatically optimized for accessibility, minimising the need for manual intervention. It’s particularly useful when producing multiple tables, as it maintains consistent formatting and accessibility standards throughout your documents.
+
+#### Producing accessible tables in Python using [``gptables``](https://pypi.org/project/gptables/) package:
+
+- Overview: The ``gptables`` package is designed to help Python users create accessible tables within spreadsheets. This package automates the formatting of tables to meet accessibility standards, ensuring that key elements such as table headers, data descriptions, and structures are correctly implemented.
+- Benefits: By using ``gptables``, you can automatically generate tables that are readable by screen readers, properly formatted for navigation, and compliant with accessibility regulations. This reduces the manual effort required to adjust table formats and enhances consistency across multiple spreadsheets.
+
+- Note on automation packages:
+Neither of these packages are guaranteed to automatically produce perfectly accessible spreadsheets. The aim is to help you automate some of the edits.
+
+#### Why Automate?
+
+- Consistency: Automation helps maintain a consistent approach to accessibility across all spreadsheets, reducing the likelihood of human error.
+- Efficiency: Automating the process saves time, especially when dealing with large datasets or generating numerous tables.
+- Compliance: These tools help ensure that your spreadsheets meet legal accessibility requirements without extensive manual adjustments, reducing the risk of non-compliance.
+
+By incorporating tools like ``gptables`` or ``a11ytables`` into your coding workflow, you can enhance the accessibility of your spreadsheets with minimal extra effort, making your data more inclusive and easier to use for everyone.
```

Comment:
Can remove this section, it's a duplication of a previous one.

Reply:
Done